charts/payment-api/values.preview.template.yaml

```diff
@@ -10,20 +10,18 @@ java:
     POSTGRES_CONNECTION_OPTIONS: "?sslmode=allow"
 
     SPRING_LIQUIBASE_ENABLED: true
-    SPRING_CLOUD_PROPERTIESVOLUME_ENABLED: false
     OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-aat2.internal:8443/openam/oauth2/hmcts
-    OIDC_CLIENT_SECRET: NVYEC2X7XFSRENTU
-    OIDC_S2S_SECRET: G5XTFNBUW4P6ZP4F
     LAUNCH_DARKLY_USER_NAME_PREFIX: aat
-
     SPRING_PROFILES_ACTIVE: "liberataMock,callbackMock,ccdMock"
     CALLBACK_PAYMENTS_CUTOFF_TIME_IN_MINUTES: 0
-
-    #Below key required for preview pipeline
-    GOV_PAY_AUTH_KEY_CMC: 8ln764q6jrbgqvvvv40uggsea3h1p602v79gdrt7ni66mrmpk416ulohvd
-    LAUNCH_DARKLY_SDK_KEY: sdk-06456c8b-312c-475c-b154-ca585f5ca063
-    CCPAY_BULK_SCANNING_API_URL: http://ccpay-bulkscanning-api-pr-251.service.core-compute-preview.internal
-    PAYMENT_S2S_SECRET: ETNY5CBN7NR62T6M
     PBA_CONFIG1_SERVICE_NAMES: PROBATE
+
+  keyVaults:
+    "ccpay":
+      resourceGroup: ccpay
+      secrets:
+        - gov-pay-keys-cmc
```

Comment:
Hi @amitchidrewar1301 - Why only these 3 secret keys are added. how about the others? We could see many in values.yaml?

Reply:
I can see currently we are using these three secrets only on preview.  We should add secrets as per need in the future.